coalib/parsing/inline_comments.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def read_ignore_comments(original_line, start_comment, end_comment=''):
+    """
+    Read Ignore Comment(s) already present in the line and
+    accordingly return a list of bears previously ignored.
```

Comment:
use `:return:` and also add metadata for the parameters that are used by sphinx to generate documentation.